README.md

```diff
@@ -1,19 +1,24 @@
-# GitOps Toolkit
+# Flux version 2
```

Comment:
I would call it Flux and update the fluxcd/flux readme and set there v1. See https://github.com/linkerd/linkerd2

Reply:
I think it's worth being explicit that it's a new thing, especially while fluxcd/flux is being maintained.

Reply:
Ok good point